Lesson2.ipynb

```diff
@@ -648,6 +589,580 @@
     "    print(\"All good!\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Advanced Types\n",
+    "\n",
+    "The advanced types in Python are also called data structures.\n",
+    "\n",
+    "A data structure is a particular way of organizing and storing data in a computer so that it can be accessed and modified efficiently. (Wikipedia)\n",
```

Comment:
We can reference the link
[https://en.wikipedia.org/wiki/Data_structure](Wikipedia)